libvast/src/command.cpp

```diff
@@ -32,10 +32,13 @@ command::~command() {
 }
 
 int command::run(caf::actor_system& sys, option_map& options,
-                 caf::message args) {
-  CAF_LOG_TRACE(CAF_ARG(options) << CAF_ARG(args));
+                 const_iterator args_begin, const_iterator args_end) {
+  CAF_LOG_TRACE(CAF_ARG(options));
   // Split the arguments.
-  auto [local_args, subcmd, subcmd_args] = separate_args(args);
+  auto args = caf::message_builder{args_begin, args_end}.move_to_message();
+  auto [local_args, subcmd, subcmd_args, consumed_args] = separate_args(args);
```

Comment:
Isn't `consumed_args` redundant, since it should always be equal to `local_args.size()`?

Reply:
Yes, it is! I will fix that.